cluster/saltbase/salt/base.sls

```diff
@@ -5,7 +5,9 @@ pkg-core:
 {% if grains['os_family'] == 'RedHat' %}
       - python
       - git
+      - glusterfs-fuse
```

Comment:
IMO there is no option to exclude glusterfs from build, so any Kubernetes release will have support for it. Gluster should be part of the default installation in the same way as e.g. nfs-utils inherited from the base image.